src/libs/README.md

```diff
@@ -3,42 +3,42 @@ elektra-libs(7) -- libs overview
 
 ## Libelektra
 
-Since [0.8.15](/doc/decisions/library_split.md) [libelektra](/src/libs/elektra)
+Since [0.8.15](/doc/decisions/library_split.md) [libelektra](elektra/)
```

Comment:
@KurtMi Why does doxygen works nice without the slash at the end? Can you make the link checker more strict that it enforces relative links if they do not need `..`?